shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.h

```diff
@@ -62,6 +63,11 @@ FLUTTER_DARWIN_EXPORT
 @property(nonatomic, copy) NSDictionary* markedTextStyle;
 @property(nonatomic, assign) id<UITextInputDelegate> inputDelegate;
 
+// Delta
+@property(nonatomic, strong) FlutterTextEditingDelta* currentTextEditingDelta;
+// Previous text editing delta for testing.
+@property(nonatomic, strong) FlutterTextEditingDelta* previousTextEditingDelta;
```

Comment:
It doesn't seem you're reading from `previousTextEditingDelta` at all. Why is it needed?

Reply:
I was using it for tests. `currentTextEditingDelta` is flushed at the end of `updateEditingStateWithDelta` so I am unable to capture the delta and verify it without this variable.

Reply:
You can have a `FlutterEngine` mock and read the json it's trying to send to the framework IIRC.

Reply:
There should be similar tests in the text input plugin test suite already for `updateEditingState`.